src/Avalonia.Controls/Platform/Win32CornerHints.cs

```diff
@@ -0,0 +1,30 @@
+using System;
+
+namespace Avalonia.Platform;
+
+/// <summary>
+/// Specifies hints for window corner appearance.
+/// </summary>
+[Flags]
+public enum Win32WindowCornerHints
+{
+    /// <summary>
+    /// Default Avalonia behavior.
+    /// </summary>
+    NoHint,
+
+    /// <summary>
+    /// The platform's default corner style.
+    /// </summary>
+    PlatformDefault,
+
+    /// <summary>
+    /// Rounded corners for the window.
+    /// </summary>
+    Rounded,
+
+    /// <summary>
+    /// Prevents corners from being rounded.
+    /// </summary>
+    DoNotRound
```

Comment:
I'm not a fan of this name. I think the whole enum should go away but "NotRounded" is probably better.

Reply:
That's right, changed it for consistency, thanks!